radsat-sk/operation/message/protobuf/proto/RFileTransfer.proto

```diff
@@ -1,37 +1,123 @@
+	uint32 mode				= 1;	///< Mode of operation at time of recording
+	uint32 uptime			= 2;	///< Uptime (in ticks) since last reset (of any kind)
+	uint32 rtcTime			= 3;	///< RTC Time (in unix time; seconds since Unix Epoch)
+	uint32 rtcTemperature	= 4;	///< 10bit Temperature recorded by OBC's RTC
+}
+
+
 
-// basic file transfer response message
-message FileTransferResponse {
-	int32 packetType = 1;	// the ID of the type of packet that is being responded to (image, log, etc.)
-	int32 packetNumber = 2;	// the specific packet number that is being responded to
-	int32 response = 3;		// the value of the response (ACK, NAK, CAN, etc.)
+// Information about the state of the Transceiver
+message TransceiverTelemetry {
+	// Telemetry for the Receiver
+	message receiverTelemetry {
+		float rx_doppler	= 1; 	///< Rx Telemetry receiver doppler.
+		float rx_rssi		= 2; 	///< Rx Telemetry rssi measurement.
+		float bus_volt		= 3; 	///< Rx Telemetry bus voltage.
+		float vutotal_curr	= 4; 	///< Tx Telemetry total current.
+		float vutx_curr		= 5; 	///< Tx Telemetry transmitter current.
+		float vurx_curr		= 6; 	///< Tx Telemetry receiver current.
+		float vupa_curr		= 7; 	///< Tx Telemetry power amplifier current.
+		float pa_temp		= 8; 	///< Rx Telemetry power amplifier temperature
+		float board_temp	= 9; 	///< Rx Telemetry board temperature.
+		uint32 uptime		= 10;	///< Rx Uptime in Seconds.
+		uint32 frames		= 11;	///< Rx Frames currently in Receive Buffer.
+	}
+
+	// Telemetry for the Transmitter
+	message transmitterTelemetry {
+		float tx_reflpwr	= 1; 	///< Tx Telemetry reflected power.
+		float tx_fwrdpwr	= 2; 	///< Tx Telemetry forward power.
+		float bus_volt		= 3; 	///< Tx Telemetry bus voltage.
+		float vutotal_curr	= 4; 	///< Tx Telemetry total current.
+		float vutx_curr		= 5; 	///< Tx Telemetry transmitter current.
+		float vurx_curr		= 6; 	///< Tx Telemetry receiver current.
+		float vupa_curr		= 7; 	///< Tx Telemetry power amplifier current.
+		float pa_temp		= 8; 	///< Tx Telemetry power amplifier temperature.
+		float board_temp	= 9; 	///< Tx Telemetry board temperature.
+		uint32 uptime		= 10;	///< Tx Uptime in Seconds.
+	}
+
+	receiverTelemetry receiver 			= 1;	///< Receiver Telemetry
+	transmitterTelemetry transmitter	= 2;	///< Transmitter Telemetry
+}
+
+// Information about the state of the Camera
+message CameraTelemetry {
+	uint32 upTime	= 1;	///< Local uptime (since last startup) in seconds
 }
 
+// Information about the state of the Electrical Power Supply
+message EpsTelemetry {
 
-// basic file transfer packet
-message FileTransferPacket {
-	int32 packetType = 1;	// the ID of the type of packet that is being sent (image, log, etc.)
-	int32 packetNumber = 2;	// the specific packet number that is being sent
-	bytes data = 3;			// the raw date bytes being transmitted
 }
 
+// Information about the state of the Battery
+message BatteryTelemetry {
+
+}
+
+// Information about the state of the Antenna
+message AntennaTelemetry {
+

```

Comment:
Add a TODO for antenna Telemetry. 

I know what the antenna telemetry is but I think it would be a better workflow to complete the PR then ill make a branch that adds all telemetry